### PR TITLE
Adding a note about a current bug.

### PR DESCRIPTION
**Pull request by sfshaza2**
Thanks for this info, @gspencergoog!

### CODE REVIEW

File: src/docs/cookbook/forms/focus.md [hunk 1]
```diff
@@ -142,6 +143,14 @@ FloatingActionButton(
 
 ## Interactive example
 
+{{site.alert.note}}
+  While the following code runs in DartPad, it fails
+  due to [Issue 52221][] when run against the 
+  stable (1.12) or beta (1.14) versions of the
+  Flutter SDK. Until this issue has landed,
```
Comment (gspencergoog) [inline src/docs/cookbook/forms/focus.md]:
```suggestion
  Flutter SDK. Until the fix for this issue has landed in the channel you are using,
```

I should probably track down what actually fixed this problem, so we can link to the PR/hash that fixes it.

Comment (sfshaza2) [reply src/docs/cookbook/forms/focus.md]: If you do, let me know, but I'm going to push it, with your edit. Thx!

**Comment by gspencergoog** [comment]
FYI, this issue was fixed in https://github.com/flutter/flutter/pull/50372, which is hash bf551a31fe7ef45c854a219686b6837400bfd94c, so once that makes it into the channel you're interested in, the example should be fine.

File: src/docs/cookbook/forms/focus.md (hunk 1 above)
Comment (sfshaza2) [reply src/docs/cookbook/forms/focus.md]: Thanks for the LGTM, @gspencergoog!

**Comment by gspencergoog** [comment]
What happened to the "fix for" in my edit?  And also, now that I found where it was fixed, we should add something that describes when it will land (give the hash where it was fixed).

**Pull request merged**